README.md

```diff
@@ -1,25 +1,31 @@
 # Sparse Coding
-Reference sparse coding implementations for efficient learning and inference implemented in PyTorch with GPU support. 
 
-## Dictionary Learning
+Reference sparse coding implementations for efficient learning and inference implemented in PyTorch with GPU support.
+
+## Features
+
+### Dictionary Learning
+
 * Repo currently includes classic patch-wise sparse coding dictionary learning.
 
-## Implemented Inference Methods
+### Implemented Inference Methods
+
 * Locally Competative Algorithm (LCA)
 * Gradient Descent with Euler's method on Laplace Prior (Vanilla)
 * Laplacian Scale Mixture (LSM)
 * Iterative Shrinkage-threshold Algorithm (ISTA)
 * Generic PyTorch minimization of arbitrary loss function (PyTorchOptimizer)
 
-# Setup
+## Setup
+
 1. Clone the repo.
 2. Navigate to the directory containing the repo directory.
-3. Run `pip install -e sparsecoding`
-4. Navigate into the repo and install the requirements using `pip install -r requirements.txt`
-5. Install the natural images dataset from this link: https://rctn.org/bruno/sparsenet/IMAGES.mat
-6. Try running the demo notebook: `examples/sparse_coding.ipynb`
+3. Run `pip install -e .[all]`
```

Comment:
To do a fresh install in zsh I had to run `pip install -e ".[all]"`, otherwise I got the error `zsh: no matches found: .[all]`.  Is that expected?

Reply:
Seems like it's a zsh thing since it interprets the brackets as special characters... annoying. Should we add a line saying "if using zsh, run `pip install -e ".[all]"`?

Reply:
Looks like other projects just use quotes:

https://github.com/google/flax/issues/3473
https://github.com/google/flax
https://github.com/jax-ml/jax/issues/14315
https://github.com/jax-ml/jax

so I propose we do that.